views/006_config_views.sql

```diff
@@ -180,3 +180,57 @@ DROP VIEW IF EXISTS config_tags;
 CREATE OR REPLACE VIEW config_tags AS
   SELECT d.key, d.value
   FROM configs JOIN json_each_text(tags::json) d ON true GROUP BY d.key, d.value ORDER BY key, value;
+
+-- config_type_summary
+DROP VIEW IF EXISTS config_type_summary;
+CREATE VIEW config_type_summary AS
+  WITH changes_per_type AS (
+    SELECT
+      config_items.config_type,
+      COUNT(config_changes.id) AS count
+    FROM
+      config_changes
+      LEFT JOIN config_items ON config_changes.config_id = config_items.id
+    GROUP BY
```

Comment:
Can we limit these to changes created in last 30 days ?

Reply:
made the change